src/components/__tests__/KeyboardShortcut.test.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react'
+import { render } from '@testing-library/react'
+import KeyboardShortcut from '../KeyboardShortcut'
+
+/**
+ * Renders the KeyboardShortcut component twice, once for macOS and non-macOS setting.
+ * @param shortcut the keyboard shortcut string
+ * @param props extra props for the KeyboardShortcut component
+ */
+function renderKeyboardShortcut(
+    shortcut: string | string[],
+    props: Omit<React.ComponentProps<typeof KeyboardShortcut>, 'children' | 'isMac'> = {},
+) {
+    const { container } = render(
+        <>
+            <KeyboardShortcut {...props} isMac={false}>
+                {shortcut}
+            </KeyboardShortcut>
+            <KeyboardShortcut {...props} isMac={true}>
+                {shortcut}
+            </KeyboardShortcut>
+        </>,
+    )
+    return container
+}
+
+describe('KeyboardShortcut', () => {
+    it('works as expected', () => {
+        const element = renderKeyboardShortcut('ctrl + alt + shift + e')
+        expect(element).toMatchSnapshot()
+    })
+
+    it('recognizes mod/cmd as a modifier that behaves differently in macOS and outside macOS', () => {
+        const element = renderKeyboardShortcut(['mod + k', 'cmd+j'])
+        expect(element).toMatchSnapshot()
+    })
+
+    it('supports passing various shortcuts, which it will separate with commas', () => {
+        const element = renderKeyboardShortcut(['p1', 'p2'])
+        expect(element).toMatchSnapshot()
+    })
+
+    it('only capitalizes a non-modifier part if the overall shortcut has modifiers', () => {
+        const element = renderKeyboardShortcut(['e', 'alt+e'])
```

Comment:
Hmmm, I'm not sure that's how it works. At least not in macOS:

![image](https://user-images.githubusercontent.com/15199/87325670-bfb16500-c4ff-11ea-9bb4-bff0d8154e2f.png)

I think it's the same in Windows and Linux but I am not 100% sure now. Maybe it's my used-to-macOS brain speaking, but in this context the capitalization of the letter is not relevant, and it tends to be capitalized all the time. Otherwise there would be no need to express `shift` in the keyboard sequence.

Reply:
Hmm good point, okay let's keep it like this for now 👍